tools/generate-component/boilerplate/component.visual.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { html } from 'lit';
+
+import { describeViewports, visualDiffStandardStates } from '../core/testing/private.js';
+
+import './__noPrefixName__.js';
+
+describe('__name__', () => {
+  /**
+   * Add the `viewports` param to test only specific viewport;
+   * add the `viewportHeight` param to set a fixed height for the browser.
+   */
+  describeViewports(() => {
```

Comment:
I think a cases with describEach example would be great. But at the end it doesn't matter. most important is, that this file is here :D